packages/mds-ingest-service/@types/index.ts

```diff
@@ -70,6 +70,19 @@ export type TimeRange = {
   start: Timestamp
   end: Timestamp
 }
+
+export const GetVehicleEventsOrderColumn = <const>['timestamp', 'vehicle_id', 'provider_id', 'vehicle_state']
```

Comment:
Just curious as I've seen this in other places as I learn typescript. Is it common to name the value and type name the same? I've seen it where the value name is descriptive and would name it like "GetVehicleEventOrderColumns" (with an s to describe these are all the columns) and the type is then without the "s". Also, is it preferred to use <const> vs "as const"? Are they the same?

Reply:
I've seen instances of both
```
export const FooTypes = <const>['bar', 'baz']
export type FooType = typeof FooTypes[number]
```

and
```
export const FooType = <const>['bar', 'baz']
export type FooType = typeof FooTypes[number]
```
In various places in the code base. I think the latter is the preferred.

My assumption is that the latter just reduces the overhead of knowing which type to choose (`FooTypes vs FooTypes`) since TS will interpret it as the type declaration. (@drtyh2o feel free to tell me I'm off base here 😄 ).

As for
`<const> foo` vs `foo as const`, they are [equivalent](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions) except in a `.tsx` file.